### DISCUSSION TITLE
Groups option is missing in Environment Variables

### DISCUSSION THREAD

**Discussion by burekas7** [codemagic-ci-cd/codemagic-docs#2381] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Hi,

Opening my app settings in Workflow UI Editor.
Opening the Environment variables,
I want to add an SSH key for Bitbucket, in the docs you say that I need also to mention the "Group" value.
But this filed isn't shown there:
<img width="900" alt="Screenshot 2024-02-18 at 2 16 36" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/13424257/96e9a268-1aeb-434e-afcf-1037c37b2fc1">

While when I go to my team account and opening the "Global variables and secrets"
The "Select group" option is there:
<img width="1302" alt="Screenshot 2024-02-17 at 21 49 20" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/13424257/6fa3d0e4-5d72-493a-8583-c6a3fc5a7224">

So why this option isn't shown also in the app settings view?

### Expected results

Selecting group option to be visible also in app settings view

### Actual results

It isn't shown there.

**Comment by dzab5** [comment]
Did you ever found the answer to this issue? 

I see this question was marked as duplicate and hence closed, but duplicate of which one @dtrdic ?

**Reply by toddmueller** [reply]
I'm having the same issues. Was this ever resolved?

**Reply by dtrdic** [reply]
@toddmueller global environment variable is a feature that is only available if you are using a codemagic.yaml configuration, and those variables have the variable group name. You can add those in Team >team name > global variables and secrets.
![Screenshot 2024-08-01 at 08 13 28](https://github.com/user-attachments/assets/90389c49-e4aa-4f16-8fe5-7cd0cca099d5)

If you are working with workflow editor then this feature is not available. But adding variables without a group name is still valid

**Reply by toddmueller** [reply]
Excellent. Thank you!
